packages/demo/src/components/examples/LimitExamples.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {Limit} from 'react-vapor';
+
+export class LimitExamples extends React.Component {
+    render() {
+        const title: string = 'Limit example';
+        const usageNumber: number = 42;
+        const usageNumberNearLimit: number = 80;
+        const usageNumberEqualToLimit: number = Limit.defaultProps.limitNumber;
+        const customLimitNumber: number = 130;
+        const customLimitTitle: string = 'Throttling limit';
+        const isLimitCanBeChanged: boolean = true;
+        const isLimitIsTheGoalToReach: boolean = true;
+        const isHistoryIsIncluded: boolean = true;
```

Comment:
I may be wrong as I'm fairly new to Typescript, but I don't think specifying types here is necessary, since you're using `const` and setting the values.